all-repos.yaml

```diff
@@ -163,3 +163,5 @@
 - https://github.com/sirosen/check-jsonschema
 - https://github.com/sirosen/fix-smartquotes
 - https://github.com/sondrelg/pep585-upgrade
+- https://github.com/ikamensh/flynt
```

Comment:
sorry, the flynt project is not welcome here -- past conduct of the maintainer is not appropriate

Reply:
Oh. I didn't know. I'll stick to vulture then.